### PR TITLE
Migrations/add priority column

### PR DESCRIPTION
**Pull request by crissthiandi**
## Summary by Sourcery

Add a 'priority' column to the 'questions' and 'question_options' tables to manage question order, and introduce 'created_at' and 'updated_at' columns with triggers to track record changes.

New Features:
- Add a 'priority' column to the 'questions' and 'question_options' tables to manage the order of questions.

Enhancements:
- Introduce 'created_at' and 'updated_at' timestamp columns to the 'questions' and 'question_options' tables for tracking record creation and updates.
- Implement triggers to automatically update the 'updated_at' column to the current timestamp on row updates in both 'questions' and 'question_options' tables.

### CODE REVIEW

**Review by tlacloc — COMMENTED** [review]
Looks good, tho it looks like the priority for question options is missing

File: hasura/migrations/academy_db/1726251123825_update_questions_columns/up.sql [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+$$ LANGUAGE plpgsql;
+CREATE TRIGGER "set_public_questions_updated_at"
+BEFORE UPDATE ON "public"."questions"
+FOR EACH ROW
+EXECUTE PROCEDURE "public"."set_current_timestamp_updated_at"();
+COMMENT ON TRIGGER "set_public_questions_updated_at" ON "public"."questions"
+IS 'trigger to set value of column "updated_at" to current timestamp on row update';
+
+alter table "public"."questions" add column "priority" integer
+ not null default '0';
```
Comment (tlacloc) [inline hasura/migrations/academy_db/1726251123825_update_questions_columns/up.sql:26]: i'll change it to 1 maybe

File: hasura/migrations/academy_db/1726256674359_add_order_in_questions_ilithya/up.sql [hunk 2]
```diff
@@ -0,0 +1,266 @@
+
+-- A shader in WebGL is...
+UPDATE questions
+SET priority = 1
+WHERE id = '76c83b70-e59f-446a-bf31-8a96b6eff6f2';
```
Comment (tlacloc) [inline hasura/migrations/academy_db/1726256674359_add_order_in_questions_ilithya/up.sql:5]: also is it only for ilithya isnt it?

Comment (crissthiandi) [reply hasura/migrations/academy_db/1726256674359_add_order_in_questions_ilithya/up.sql:5]: yes, only arrange ilithy's questions

File: hasura/migrations/academy_db/1726251123825_update_questions_columns/up.sql (hunk 1 above)
Comment (crissthiandi) [reply hasura/migrations/academy_db/1726251123825_update_questions_columns/up.sql:26]: I try to use this startegy: set 0 like a no setted, and use 1,2,3.. for priority setted, what do you think?

File: hasura/migrations/academy_db/1726251123825_update_questions_columns/up.sql [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+$$ LANGUAGE plpgsql;
+CREATE TRIGGER "set_public_question_options_updated_at"
+BEFORE UPDATE ON "public"."question_options"
+FOR EACH ROW
+EXECUTE PROCEDURE "public"."set_current_timestamp_updated_at"();
+COMMENT ON TRIGGER "set_public_question_options_updated_at" ON "public"."question_options"
+IS 'trigger to set value of column "updated_at" to current timestamp on row update';
+
+alter table "public"."question_options" add column "priority" integer
+ not null default '0';
```
Comment (crissthiandi) [inline hasura/migrations/academy_db/1726251123825_update_questions_columns/up.sql:52]: yes, we add the columnd here
https://github.com/Blind-Gallery/BG-Academy/pull/455#pullrequestreview-2304248962

**Review by tlacloc — APPROVED** [review]

**Pull request merged**